dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java

```diff
@@ -119,8 +122,12 @@ private void stopChannel(String name, Event event) {
         try {
             ChainData[] chains = cf.getAllChains(name);
             for (ChainData chain : chains) {
+
                 if (cf.isChainRunning(chain)) {
-                    // stop the chain now.. 
+                    if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                        Tr.event(tc, "Chain running; channel=" + name);
+                    }
```

Comment:
Do you want to log the actual chain name here? Something like `chain.getName()`.

We already look to log the name on entry to this method, then use the name to get all the chains and then loop through them to check if those chains are running and if so perform some actions on them.

Reply:
I agree, line 119 in this PR will log the channel name, perhaps the new log should log the current iteration ChainData's object name .

Reply:
I'll remove it since isChainRunning()
 logs the name.